cmd/validate/image.go

```diff
@@ -402,12 +403,10 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 		"[DEPRECATED] write output to a file. Use empty string for stdout, default behavior")
 
 	cmd.Flags().BoolVarP(&data.strict, "strict", "s", data.strict,
-		"return non-zero status on non-successful validation")
+		"Return non-zero status on non-successful validation. Defaults to true. Use --strict=false to return a zero status code.")
```

Comment:
@simonbaird Is this better

Reply:
comment from simonbarid: Yeah, that should probably be updated a little to indicate the change. Also this part:

404     cmd.Flags().BoolVarP(&data.strict, "strict", "s", data.strict,                                                                                
405         "return non-zero status on non-successful validation")   
Something like "return non-zero status on non-successful validation. Defaults to true. Use --strict=false to return a zero status code" or similar in both places.